app/webpacker/controllers/vertical_ellipsis_menu_controller.js

```diff
@@ -0,0 +1,21 @@
+import { Controller } from "stimulus";
+
+export default class extends Controller {
+  static targets = ["content"];
+
+  connect() {
+    super.connect();
+    window.addEventListener("click", (e) => {
+      if (this.element.contains(e.target)) return;
+      this.#hide();
+    });
```

Comment:
🤖 Rubocop would say to have an empty line after a guard clause  ;)

```suggestion
    window.addEventListener("click", (e) => {
      if (this.element.contains(e.target)) return;

      this.#hide();
    });
```

Hmm I'd also suggest we don't embed functions for event listeners. Moving it to another function gives us a chance to make it easy to read:


```suggestion
    window.addEventListener("click", this.#hideIfClickedOutside);
```

Reply:
I agree. Will do.